docs/shards.txt

```diff
@@ -1,5 +1,101 @@
-.. _shards:
+.. _shards-browser:
 
 ==============
 Shards Browser
 ==============
+
+The CrateDB admin UI comes with a *shards browser* that provides you with a
+visual overview of all the `shards`_ in your cluster.
+
+.. NOTE::
+
+   The shards browser is an `enterprise feature`_.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Screenshot
+==========
+
+Here's what a simple database might look like:
+
+.. image:: shards-simple.png
+
+In this example, there are two primary columns:
+
+- The *blob* column holds `BLOB tables`_
+
+- The *doc* column holds regular `document tables`_
+
+The *blob* column lists a single table:
+
+- ``my_blobs``
+
+The *doc* column lists two tables:
+
+- ``another_table``
+- ``tweets``
+
+The table has a single row corresponding to the one node in this cluster.
+CrateDB automatically names unnamed nodes. In this instance, there is a single
+node named *Monte Civetta*.
+
+In this example, the *Monte Civetta* node is holding the following shards:
+
+- Three shards for the table named ``my_blobs``
+- Four shards for the table named ``another_table``
+- Four shards for the table named ``tweets``
+
+Shards are colored to indicate their status and there is a color key at the
+top of the table.
+
+The color of the shards in this example indicates that they are all primary
+shards and have been successfully started.
+
+Features
+========
+
+.. |nbsp| unicode:: 0xA0
+   :trim:
+
+:Shard |nbsp| ID |nbsp| display:
+
+  You can toggle the display of shard IDs by selecting or unselecting the *Show
+  Shard IDs* checkbox.
+
+:Shard |nbsp| status:
+
+  Each shard is colored according to its status:
+
+  :Started |nbsp| Primary: (bright green)
+
+     This is a primary shard and has been successfully started.
+
+  :Started |nbsp| Replica: (dark green)
+
+     This is a `replica`_ shard and has been successfully started.
+
+  :Initializing: (dark yellow)
+
+     The shard is being initialized (i.e. started).
+
+     FIXME: is this true?
+
+  :Relocating: (dark red)
+
+     The shard is being relocated to another node.
+
+     CrateDB automatically rebalances your cluster for you. While a shard is
+     being moved to another node, it is unavailable for use.
+
+  :Unassigned: (gray)
+
+     FIXME: what does this mean?
```

Comment:
@mxm here

Reply:
A shard that has not been assigned to a node. Effectively, if the shard has any data, it won't be accessible in the cluster until it becomes started.

Reply:
Yes, they are two separate concepts. The shard's physical data can be recovered but it might not yet be accessible via cluster routing.